netstandard/ref/System.cs

```diff
@@ -3814,7 +3812,6 @@ public enum StringComparison
         Ordinal = 4,
         OrdinalIgnoreCase = 5,
     }
-    [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public static partial class StringNormalizationExtensions
     {
```

Comment:
Presumably we want to keep hiding this one as the type was a .NET Core 1.x helper that is now replaced with the actual instance method.

Reply:
Kept.